autoload/slime.vim

```diff
@@ -107,7 +107,7 @@ endfunction
 
 function! s:NeovimConfig() abort
   if !exists("b:slime_config")
-    let b:slime_config = {"jobid": "3"}
+    let b:slime_config = {"jobid": g:slime_jobid}
```

Comment:
what's the behavior here if `g:slime_jobid` is undefined?

Reply:
You get a prompt:

> Error detected while processing function slime#send_op[1]..<SNR>124_SlimeGetConfig[14]..<SNR>124_SlimeDispatch[6]..<SNR>124_NeovimConfig:                                                                                            
> line    2:
> E121: Undefined variable: g:slime_jobid

After enter:

> Error detected while processing function slime#send_op[1]..<SNR>124_SlimeGetConfig[14]..<SNR>124_SlimeDispatch[6]..<SNR>124_NeovimConfig:
> line    2:
> E15: Invalid expression: {"jobid": g:slime_jobid}

Clearly indicating you should read the README again...

Reply:
hi @HTransistor,

although I agree the previous behavior (default to magic number `3`) wasn't great, I don't think this is a friendly workaround.

I think NeovimConfig could use a completion function like the Screen and Tmux integrations… but since I'm not familiar with the Neovim mechanism about assigning (and enquiring about) jobids, I'm not in a good position to write it.

Reply:
I don't know why you think, that leaving it broken for everybody who want to use neovim, is better, but okay.